data/basics/userBalance_test.go

```diff
@@ -190,8 +190,11 @@ func TestEncodedAccountDataSize(t *testing.T) {
 		}
 		maxLocalState[maxKey] = maxValue
 	}
-
-	for appCreatorApps := 0; appCreatorApps < currentConsensusParams.MaxAppsCreated; appCreatorApps++ {
+	maxAppsCreate := currentConsensusParams.MaxAppsCreated
+	if maxAppsCreate == 0 {
+		maxAppsCreate = config.Consensus[protocol.ConsensusV30].MaxAppsCreated
```

Comment:
wouldn't this revert an unlimited MaxAppsCreated to 10?

Reply:
Oh I see you are using this to generate test data, so just make 10 apps if apps are unlimited.

Reply:
exactly.